licenses/bin/ua-parser-js.MIT

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2012-2018 Faisal Salman <<f@faisalman.com>>
+Copyright (c)
```

Comment:
This looks wrong. There should be a year and copyright holder.

Reply:
Hmm, I thought I removed this file. This is not needed since `ua-parser-js` is not bundled. Will remove.